src/resource/createContainerInContainer.ts

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright 2022 Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import {
+  UrlString,
+  createContainerInContainer as coreCreateContainerInContainer,
+  WithResourceInfo,
+  SolidDataset,
+} from "@inrupt/solid-client";
+import { VerifiableCredential } from "@inrupt/solid-client-vc";
+import { fetchWithVc } from "../fetch";
+import { FetchOptions } from "../type/FetchOptions";
+
+/**
+ * Create an empty Container inside the Container at the given URL.
+ *
+ * Throws an error if creating the Container failed, e.g. because the current user does not have
+ * permissions to.
```

Comment:
```suggestion
 * Throws an error if creating the Container failed, e.g. because the current user does not have
 * permissions to. In particular, the Access Grant being used should at least append access to the target Container.
```